public/main/inc/lib/webservices/Rest.php

```diff
@@ -1432,6 +1435,97 @@ public function subscribeUserToCourse($params)
         return [true];
     }
 
+     /**
+     * @param $course_id
+     * @return array
+     */
+    public function getHomework($course_id){
```

Comment:
Please use only camelCase variable names for new code. lowercase with _ is only for legacy code.